app/assets/stylesheets/global.sass

```diff
@@ -1496,25 +1496,66 @@ section#static_page
     padding: 60px 0
 
 
+body.modal-open .body-container
+  filter: blur(5px)
+
 .modal
   text-align: center
   .close
     i
-      font-size: 20px
+      font-size: 30px
 .modal-dialog
   text-align: left
   display: inline-block
-  width: auto
-  margin-top: 90px
-  // top: 50%
-  // position: absolute
+  width: 100%
+  margin: 22px 0 0
+
+  @media (min-width: $screen-sm-min)
+    margin: 60px 0 0
+
+  &:before
+    background-image: image-url("mudamos-logo.svg")
```

Comment:
you can add `filter: drop-shadow(0 0 14px rgba(0, 0, 0, 0.12)) drop-shadow(0 14px 14px rgba(0, 0, 0, 0.24))` to set the shadow here (by using `filter` the shadow ignores the empty pixels), maybe you'll need to put the image right on `content`